src/test/java/com/google/cloud/discotoproto3converter/DiscoToProto3ConverterAppTest.java

```diff
@@ -46,6 +46,26 @@ public void convert() throws IOException {
     assertEquals(baselineBody, actualBody);
   }
 
+  @Test
+  public void convertWithIgnorelist() throws IOException {
```

Comment:
Please consider passing as ignored most of the stuff in the generated  `compute.proto` file to reduce the size of the baseline file. Huge baseline files are painful to maintain and are maintaining, keeping them shorter is usually a good idea. (i.e. like leave only 1 message there)

Reply:
I've ignored everything but `Addresses`